src/Symfony/Component/DependencyInjection/ChildDefinition.php

```diff
@@ -162,11 +162,21 @@ public function setDeprecated($boolean = true, $template = null)
     /**
      * {@inheritdoc}
      */
-    public function setAutowiredMethods(array $autowiredMethods)
+    public function setAutowiredCalls(array $autowiredCalls)
```

Comment:
@dunglas @stof if you're ok with this rename, I can submit a refactoring PR first, to reduce the diff of this PR, OK for you?